metaci/api/serializers/testmethod_perf.py

```diff
@@ -0,0 +1,15 @@
+from rest_framework import serializers
+from collections import OrderedDict
+
+
+class NonNullModelSerializer(serializers.Serializer):
+    def to_representation(self, instance):
+        result = super(NonNullModelSerializer, self).to_representation(instance)
+        return OrderedDict(
+            [(key, result[key]) for key in result if result[key] is not None]
+        )
+
+
+class TestMethodPerfSerializer(serializers.Serializer):
+    def to_representation(self, instance):
+        return instance
```

Comment:
Does this actually work, or is it just a placeholder?

Reply:
Can you take a look at the file now and see if it is clearer please?

https://github.com/SFDO-Tooling/MetaCI/blob/06ee51b2b9625c4cbab2d86b343e7fc5aa6746b7/metaci/api/serializers/simple_dict_serializer.py

Reply:
Yeah, the part I was missing is that the values method of a queryset makes it return a dict rather than model isntances.